roles/ceph-rgw-loadbalancer/tasks/pre_requisite.yml

```diff
@@ -19,7 +19,7 @@
 
 - name: set_fact vip to vrrp_instance
   set_fact:
-      vrrp_instances: "{{ vrrp_instances | default([]) | union([{ 'name': 'VI_' + index|string , 'vip': item, 'master': groups[rgwloadbalancer_group_name][index] }]) }}"
+      vrrp_instances: "{{ vrrp_instances | default([]) | union([{ 'name': 'VI_' + index|string , 'vip': item }]) }}"
   loop: "{{ virtual_ips | flatten(levels=1) }}"
   loop_control:
       index_var: index
```

Comment:
Do we still need the `loop_control` because we don't use `index` anymore ?

Reply:
I believe `index` is still referenced in the `keepalived.conf.j2` template: `virtual_router_id {{ 50 + loop.index }}`

Reply:
Looks like we no longer require it.
It's now removed.

Reply:
No longer necessary.. removed.

Reply:
No longer necessary.. removed.